docs/how-to-use-proof.md

```diff
@@ -0,0 +1,25 @@
+# A Guide on How to Use Asset Proofs in Scalar DL
+
+This document sets out some guidelines for using Asset Proofs in Scalar DL.
+
+## What is an Asset Proof in Scalar DL ?
+
+An Asset Proof in Scalar DL is a set of information about an asset record that is created along with the asset. It is composed of the following records so that it can be treated as evidence of creation of an asset record by a ledger.
+
+- ID of an asset record
+- Age of the asset record
+- Nonce of the execution request that creats the asset record
+- Cryptographic hash of the asset record
+- Digita sigunature of the above four records 
+
+## The Benefits of Asset Proof
+
+Since Asset Proofs are evidence at the time of exucution by a ledger, it is hard for the leger to make a lie (tamper data) after the evidence is created because the proofs and the ledger states would be inconsistent.
```

Comment:
```suggestion
Since Asset Proofs are evidence at the time of execution by a ledger, it is hard for the leger to make a lie (tamper data) after the evidence is created because the proofs and the ledger states would be inconsistent.
```